src/components/Reviews.jsx

```diff
@@ -33,15 +38,25 @@ const Reviews = () => {
 
     return (
         <div>
+            <section className="filterBar">
+                <select value={sortBy} onChange={(e) => setSortBy(e.target.value)}>
+                    <option value="created_at">Date</option>
+                    <option value="comment_count">Number of Comments</option>
+                    <option value="votes">Number of Votes</option>
+                </select>
+                <select value={orderBy} onChange={(e) => setOrderBy(e.target.value)}>
+                    <option value="asc">Ascending</option>
+                    <option value="desc">Descending</option>
+                </select>
+            </section>
             <ol>
                 {reviews.map((review) => {
                     return (
                         <li className="review" key={review.review_id}>
                             <Link to={`/reviews/${review.review_id}`} className="text-link">
                                     <h2>{review.title}</h2>
                             </Link>
-                            <p>By: {review.owner}</p>
-                            <p>{review.category}</p>
+                            <p>By: {review.owner} | {moment(review.created_at).fromNow()}</p>
```

Comment:
This is a really cool way of displaying the time, I've not seen anyone do it like this but looks awesome!